packages/core/src/Dropdown/stories/Dropdown.stories.tsx

```diff
@@ -213,3 +230,20 @@ export const Virtualized: StoryObj<HvDropdownProps> = {
   decorators: [widthDecorator],
   render: () => <VirtualizedStory />,
 };
+
+export const CustomDropdown: StoryObj<HvBaseDropdownProps> = {
+  parameters: {
+    docs: {
+      description: {
+        story:
+          "You can also use the `HvBaseDropdown` component to create your own dropdown components. This component can be leveraged to create similar dropdown components to the ones we provide, or you can use the `component` prop to add it to any component of your choosing.",
+      },
+      source: {
+        code: CustomDropdownRaw,
+      },
+    },
+    eyes: { include: false },
```

Comment:
We can also remove this. It's no longer needed. All stories are disabled in Applitools by default now.